redcap_bridge/project_building.py

```diff
@@ -160,6 +165,15 @@ def combine_values(value1, value2):
     if output_file is not None:
         combined_df.to_csv(output_file, index=False)
 
+    # print number of customizing field
+    print("\nCustomizing", len(customization_df), "fields\n", end='')
```

Comment:
```suggestion
    print(f"\nCustomizing: {len(customization_df)} fields\n", end='')
```

Reply:
This is a modern (python>3.3 ?) version of formatting strings. I think it's more readable. @killianrochet What do you think?

Reply:
Yes, It's more readable, I will go on that